fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -56,11 +59,13 @@ class ExponentiatedGradient(BaseEstimator, MetaEstimatorMixin):
         (defaults to `sample_weight`)
     """
 
-    def __init__(self, estimator, constraints, eps=0.01, max_iter=50, nu=None,
+    def __init__(self, estimator, constraints, objective=None,
```

Comment:
Since this breaks positional args anyway (after `constraints` at least), do you want to add a `*` here?

Reply:
I can put `*` before `objective`, but probably not any earlier (at least not in this PR). Sounds good?

Reply:
That's what I meant - adding `objective` breaks everything _after_ `constraints` as a positional arg.